# no request data required

---

# also no return data, because graph will be published on corresponding topic